openftc/src/main/java/org/openftc/openftc/OpenFTCConfig.java

```diff
@@ -1,4 +1,4 @@
-package com.openftc;
+package org.openftc.openftc;
```

Comment:
Why expand the package name? I get the change from com.openftc to org.openftc, but why add another layer?